src/backend/distributed/commands/utility_hook.c

```diff
@@ -84,13 +97,13 @@ static bool IsDropSchemaOrDB(Node *parsetree);
 
 
 /*
- * CitusProcessUtility is a convenience method to create a PlannedStmt out of pieces of a
+ * ProcessUtilityForNode is a convenience method to create a PlannedStmt out of pieces of a
```

Comment:
maybe `ProcessUtilityCommandForNode`

Reply:
(also `ForNode` part sounds a bit confusing to me)

Reply:
Postgres doesn't seem to have `command` terminology for this purpose?